automation/check-patch.e2e-k8s.sh

```diff
@@ -14,7 +14,7 @@ teardown() {
 }
 
 main() {
-    export KUBEVIRT_PROVIDER='k8s-1.23'
+    export KUBEVIRT_PROVIDER='k8s-1.24'
```

Comment:
> why not 1.25 ? or even 1.26 centos8 one (but it is less used atm on kubevirt,
and centos9 is too new so make sense to be on 1.25)

I prefer to keep compatibility with the oldest possible version, and ship only to that. 

> I think it worth to also have here
export KUBEVIRT_PROVIDER="${KUBEVIRT_PROVIDER:-k8s-1.24}"
not just export, less suprises because it wont override if exists
and will allow to override if needed

that's the automation script invoked by CI. User's aren't meant to run that. 

No one's overriding anything there.

Reply:
@oshoval ^

Reply:
IMO you should have used the [make target](https://github.com/kubevirt/macvtap-cni/blob/7a3e9f2d39e4ce71a96cccb68e2362d2b51647dc/Makefile#L97) to run the tests, not the automation script used by CI.

That script is a one trick pony, using a set environment. It is even named `check-patch` .

Is something preventing you from using the make target ?

Reply:
nothing preventing
but for coverage, if i want to be as close to what CI is doing
just my opinion anyhow